pkg/experimental/tests/rpm/tree_artifacts/rpm_contents_vs_manifest_test.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import csv
+import io
+import os
+import rpm_util
+from rules_python.python.runfiles import runfiles
+
+EXPECTED_RPM_MANIFEST_CSV = """
+path,digest,user,group,mode,fflags,symlink
+/test_dir/a,dc35e5df50b75e25610e1aaaa29edaa4,root,root,100644,,
+/test_dir/b,dc35e5df50b75e25610e1aaaa29edaa4,root,root,100644,,
+/test_dir/subdir/c,dc35e5df50b75e25610e1aaaa29edaa4,root,root,100644,,
+/test_dir/subdir/d,dc35e5df50b75e25610e1aaaa29edaa4,root,root,100644,,
+/test_dir/e,d41d8cd98f00b204e9800998ecf8427e,root,root,100644,,
+/test_dir/f,d41d8cd98f00b204e9800998ecf8427e,root,root,100644,,
+""".strip()
+
+
+class PkgRpmCompManifest(unittest.TestCase):
+    def setUp(self):
+        self.runfiles = runfiles.Create()
+        self.maxDiff = None
+        # Allow for parameterization of this test based on the desired RPM to test.
+        self.rpm_path = self.runfiles.Rlocation(os.path.join(
```

Comment:
I think you want '/'.join().   What I found when trying to make tests portable to windows is that Rlocation insists on Bazel paths, not os specific ones.  I  had to replace all my carefully thought out os.path uses with /.

Reply:
Right, I remember this.  Fixed.

We might want to consider using something like `PosixPath` from [pathlib] to assemble bazely paths in the future.

pathlib: [https://docs.python.org/3/library/pathlib.html#pathlib.PurePosixPath]